### ISSUE TITLE
Select/Deselect All option on tables not accessible by keyboard or screenreader

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks#1647]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

Select/Deselect All options in table headers are not accessible by keyboard or screen reader. 

Tables for Front Matter, Back Matter, and Parts on the Organize page allow users to select or deselect all posts when the following column headers are clicked: 

- Show in Web
- Show in Exports
- Show Title 

Table headers on the Organize page have role=”button”. Firstly, the button role should be added to an element, such as a span, nested within the table header, rather than the <th> element itself. Currently these header cells are read as blank or skipped by the screen reader (depending on the browser), likely because the button role overrides the semantic information of the table header. 

Try: 
<th>
<span role="button" tabindex="0">Show in Web</span>
</th>

**Notes:** A tabindex attribute has also been added in the code example above. Because <th> elements are not keyboard accessible by default, these buttons also do not currently receive keyboard focus (see 2.1.1). 
Secondly, the purpose of these buttons (check/uncheck all pages in column) isn’t clear, and additional information through the use of an ARIA label or other method should clarify what pressing the button does. For example add an aria-label=”check/uncheck Show in Web for all Front Matter pages”. 
Lastly, the state of these buttons (pressed or not pressed) is not specified, and so screen reader users have no way of telling whether or not column items are checked or unchecked. Add an aria-pressed attribute to indicate if the button has been pressed. For example, if some buttons within the table are checked while others are not, the attribute should be aria-pressed=”undefined”. If all boxes have been checked, it should be set to aria-pressed=”true”, and if unchecked, aria-pressed=”false. Ideally the label attribute should change dynamically depending on the state of the buttons within the table.
Alternatively, an approach that places a checkbox in the header itself as well for selecting all items, may be another way of presenting the same capabilities.

**Comment by SteelWagstaff** [comment]
Already covered by #1633
